packages/fxa-settings/src/pages/Authorization/container.tsx

```diff
@@ -85,15 +85,15 @@ const AuthorizationContainer = ({
         session
       );
 
-      if (error === AuthUiErrors.SESSION_EXPIRED) {
+      if (error?.errno === AuthUiErrors.INVALID_TOKEN.errno) {
         throw new OAuthError('PROMPT_NONE_NOT_SIGNED_IN');
       }
 
       if (error) {
         throw error;
       }
 
-      if (!data?.verified) {
+      if (data?.verified === false) {
```

Comment:
Before this change, we throw if `data` is undefined. With this change, we only throw if data is defined and `verified` is `false`. Is that intentional?